lib/isolate_runner.dart

```diff
@@ -7,9 +7,9 @@ library isolate.isolate_runner;
 import "dart:async";
 import "dart:isolate";
 
-import 'ports.dart';
-import 'runner.dart';
-import 'src/lists.dart';
+import "ports.dart";
+import "runner.dart";
+import "src/util.dart";
```

Comment:
Should `util.dart` be checked in into the repo?

Reply:
That would be a wise move, yes. Thanks :)